bin/install

```diff
@@ -114,7 +114,12 @@ get_platform() {
 
 get_arch() {
   local -r machine="$(uname -m)"
-  OVERWRITE_ARCH=${ASDF_HASHICORP_OVERWRITE_ARCH:-"false"}
+  local -r upper_toolname="$(echo ${toolname} | tr '[:upper:]' '[:lower:]')"
```

Comment:
The lint check is quite unhappy about this line! It might be easiest to pull it out into a separate function.

Reply:
Can do!